core/src/main/java/org/apache/carbondata/core/datamap/Segment.java

```diff
@@ -80,6 +115,22 @@ public static Segment toSegment(String segmentId) {
     return new Segment(segmentId, null);
   }
 
+  /**
+   * SegmentId can be combination of segmentNo and segmentFileName
```

Comment:
can you describe more what is in the SegmentId

Reply:
Done